src/HealthGPS.Datastore/datamanager.cpp

```diff
@@ -1,33 +1,86 @@
 #include "datamanager.h"
 #include "HealthGPS.Core/math_util.h"
 #include "HealthGPS.Core/string_util.h"
+#include "HealthGPS/program_path.h"
 
 #include <fmt/color.h>
 #include <fstream>
+#include <jsoncons/json.hpp>
+#include <jsoncons_ext/jsonschema/jsonschema.hpp>
 #include <rapidcsv.h>
 
 #include <utility>
 
+namespace {
+using namespace jsoncons;
+
+json resolve_uri(const jsoncons::uri &uri, const std::filesystem::path &schema_directory) {
+    constexpr const char *url_prefix = "http://example.com/";
```

Comment:
Yes, exactly. Technically, we don't *have* to serve them from anywhere (we could just use local copies of the files), but I guess it's probably best practice to publish the schemas so people can validate files using external tools etc.

I think it'll be fine to serve them from GitHub. I guess we could either just use URLs to the files in the repo or we could publish them with the documentation.

Reply:
Yeah, exactly. I think we should still bundle the latest versions of the schemas with Health-GPS itself and read them locally, but it would be nice if we could provide proper URLs to those files too.